### ISSUE TITLE
Centos 7.5: librabbitmq-tools dep no longer valid?

### ISSUE THREAD

**Issue by samjaninf** [ONLYOFFICE/DocumentServer#305]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
onlyoffice-Document server fails to install citing a missing dependency.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Follow the directions at ```https://helpcenter.onlyoffice.com/server/linux/document/linux-installation-centos.aspx```

When you reach this step: 
```sudo yum install onlyoffice-documentserver```

The failure will occur.

**What is the expected behavior?**
onlyoffice-DocumentServer should install completely.

**Did this work in previous versions of DocumentServer?**
I assume yes.

**DocumentServer version:**
sudo yum install onlyoffice-documentserver

**Operating System:**
CentOS 7.5

**Browser version:**
Google Chrome Version 66.0.3359.117 (Official Build) (64-bit)

````yum install -y onlyoffice-documentserver
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: centos.mirrors.tds.net
 * epel: mirror.grid.uchicago.edu
 * extras: centos.mirror.constant.com
 * updates: mirror.cc.columbia.edu
Resolving Dependencies
--> Running transaction check
---> Package onlyoffice-documentserver.x86_64 0:5.1.3-35 will be installed
--> Processing Dependency: librabbitmq-tools for package: onlyoffice-documentserver-5.1.3-35.x86_64
--> Finished Dependency Resolution
Error: Package: onlyoffice-documentserver-5.1.3-35.x86_64 (onlyoffice)
           Requires: librabbitmq-tools
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
````

It appears that the librabbitmq-tools package was remove and it go merged into another package. I have also noticed that the tools provided by that package are provided in the librabbitmq-examples package.

````
yum provides /usr/bin/amqp-consume
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: centos.mirrors.tds.net
 * epel: mirror.grid.uchicago.edu
 * extras: mirror.genesisadaptive.com
 * updates: mirror.atlanticmetro.net
librabbitmq-examples-0.8.0-2.el7.x86_64 : Examples built using the librabbitmq
Repo        : base
Matched from:
Filename    : /usr/bin/amqp-consume



librabbitmq-examples-0.8.0-2.el7.x86_64 : Examples built using the librabbitmq
Repo        : @base
Matched from:
Filename    : /usr/bin/amqp-consume
````

**Comment by raphson93** [comment]
@samjaninf is librabbitmq-tools installed before the document server ?

**Comment by ShockwaveNN** [comment]
Seems by https://community.nethserver.org/t/its-time-to-test-centos-7-5/9743/4 and `librabbitmq-0.8.0-2.el7` replaced it, ok we try to figure something out

**Comment by raphson93** [comment]
#297

**Comment by samjaninf** [comment]
@raphson93 No. Following the directions, librabbitmq-tools would get installed as a dependency of documentserver, but if you try to install it on this current version of CentOS that package does not exist inside the epel repo anymore. I thought about getting the package from elsewhere but it seemed easier to just use the debian install which worked flawlessly.

**Comment by ShockwaveNN** [comment]
We removed dependency https://github.com/ONLYOFFICE/document-server-package/commit/3082096a9fb62088dcc1a54a6d9465d8349b7a61 in and it will be released as 5.1.4
@samjaninf btw we highly recommend using docker version, but if deb is ok for you - it's should be stable also

**Comment by samjaninf** [comment]
Awesome! I looked at the docker version but as I was already running them off of oVirt it seems sort of overkill on the virtualization. So I just went with the CentOS version as just about all we use inhouse is CentOS.
But yeah the Debian version is working well for now as is.
Thanks!

**Comment by ShockwaveNN** [comment]
5.1.4 released, this issue is fixed
